timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -472,8 +468,9 @@ def is_quantile_model(self) -> bool:
 
     def _get_model_params(self) -> dict:
         model_params = super()._get_model_params()
-        model_params.setdefault("scaler", "mean_abs")
-        model_params.setdefault("differences", [])
+        model_params.setdefault("target_scaler", "mean_abs")
+        if "differences" not in model_params or model_params["differences"] is None:
+            model_params["differences"] = []
```

Comment:
This change is needed so that `{"differences": None}` is interpreted as "auto differences". Currently (without the fix), `{"differences": None}` raises an error.

Reply:
What does "auto differences" mean?

Reply:
"auto differences" = we set differences to `[get_seasonality(model.freq)]` by default.